lib/mayaUsd/render/MaterialXGenOgsXml/CMakeLists.txt

```diff
@@ -68,3 +68,25 @@ install(FILES ${NODE_DECLARATIONS}
 install(FILES ${NODE_GLSL_IMPLEMENTATIONS}
     DESTINATION ${CMAKE_INSTALL_PREFIX}/libraries/adsk/maya/genglsl
 )
+
+if(MaterialX_VERSION VERSION_LESS "1.38.4")
+    MESSAGE(STATUS "Using 1.38.3 libraries with MaterialX ${MaterialX_VERSION}")
```

Comment:
Handled the library updated by backing-up the 1.38.3 versions into their own folder. The versions in the libraries folder are at the latest level.